library/VTKExtensions/Rendering/vtkF3DInteractorStyle.cxx

```diff
@@ -114,7 +114,19 @@ void vtkF3DInteractorStyle::Dolly()
   {
     return;
   }
-  this->Superclass::Dolly();
+
+  vtkRenderWindowInteractor* rwi = this->Interactor;
+  const double* center = this->CurrentRenderer->GetCenter();
+  const int dy = rwi->GetEventPosition()[1] - rwi->GetLastEventPosition()[1];
+  const int dx = rwi->GetEventPosition()[0] - rwi->GetLastEventPosition()[0];
+  auto dt = std::abs(dy) > std::abs(dx) ? dy : dx;
```

Comment:
@Meakk, are we ok with `auto` now?

Reply:
using auto does not seem needed in that case ? 

I meant that it definitely can be used when needed ^^